daos-cortx/src/daos_kv_benchmark/README.md

```diff
@@ -0,0 +1,16 @@
+This directory contains 3 files.
```

Comment:
please mention prerequisites to use google benchmark and link to how to build and install.

Reply:
done